test/service.thrift

```diff
@@ -9,8 +9,13 @@ service Foo {
     Struct returnsStruct()
 }
 
+service Qux extends Foo {
+    void quux()
+}
```

Comment:
Place this definition before `Foo` to make the test fail due to a forward reference.